Update devices-wipe.md (#3918)
MicrosoftDocs/memdocs: memdocs/intune/remote-actions/devices-wipe.md

1. Sign in to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
2. Choose **Devices** > **All devices** > choose the device > **Retire**.
3. Visit [business.apple.com](http://business.apple.com) and search for the device by its serial number.

> 4. Select the Device , choose **Release from Organization**.

Reviewer: ```suggestion
4. Select the device and choose **Release from Organization**.
```

> 4. Select the Device , choose **Release from Organization**.

Reviewer 2: ```suggestion
4. Select the device, and then select **Release from Organization**.
```

> 5. Check **I understand this cannot be undone** and click on **Release**

Reviewer: ```suggestion
5. Check **I understand this cannot be undone** and select **Release**.
```

> 5. Check **I understand this cannot be undone** and click on **Release**

Reviewer 2: ```suggestion
5. Check **I understand this cannot be undone**, and then select **Release**.
```
